packages/app/src/cli/services/app/config/link.test.ts

```diff
@@ -560,6 +560,80 @@ embedded = false
     })
   })
 
+  test('fetches the privacy compliance webhooks from the configuration module', async () => {
```

Comment:
@alvaro-shopify 👋  I'm taking over Gonzalo's PR (mostly just taking care of merge conflicts), I noticed that you completely deleted [a similar test from this file in this PR on line ~538](https://github.com/Shopify/cli/commit/ea4983751517bc7f5490787afa1dfe46d6a09f98#diff-09c68a3f5fbcd7cb3ddbe117fc5c4876b7453f2632ac812690640f774b4ac85a) - what are your thoughts on keeping this test?